src/components/ComponentSettingsModal.jsx

```diff
@@ -170,7 +170,7 @@ const ComponentSettingsModal = ({
         setErrors((prevState) => ({
             ...prevState,
             [id]: errorMessage,
-            title: titleErrorMessage,
+            title: (id === 'title' || id === 'date') ? titleErrorMessage : prevState.title
         }));
```

Comment:
This still doesn't work correctly - having a title error, followed by a date error will remove the title error. 

Also for resources, the duplicate file name check never comes into play - since the titles are compared against the actual file name (which has a date-type-title format) in `pageFileNames.filter(file => file !== fileName)`, we never actually encounter duplicates.

Should we be allowing for identical titles if the date is different?

Reply:
Discussed offline, we should disable files from having identical names even if they are of different file types or on posted on different dates.

Reply:
Fixed in 7f76d17